File: Basic/PrimeNumbers/PrimeNumbers.c
#include <stdio.h>

int main()
{
    int num,r;
    printf("Enter a number: ");
    scanf("%d",&num);
    for(int i=2;i<num;i++)
    {
        r=num%i;
        if(r==0)
        {
            printf("The number you have entered is not prime.\n");
            return 0;
        }
    }

    printf("The number you have entered is prime.\n");
    return 0;
}

File: Basic/PrimeNumbers/PrimeNumbers.py
def main():

    num = int(input("Enter a number: "))

    for i in range(2,num):
        if num % i == 0:
            print("The number is not prime.")
            return 0
    print("The number is prime.")
    return 0


if __name__ == '__main__':
    main()
